From fe7e37eb640e4e1af021ff3b03c24e6abb87ee50 Mon Sep 17 00:00:00 2001
From: arun <arun.radhakrishnan@intel.com>
Date: Thu, 24 Mar 2016 18:00:30 -0600
Subject: [PATCH 11/11] iio: Adding support hrtimer support for iio

The CHT_MRD Magnetometer(ak09911) does not have an interrupt pin
connected and the polling method does not seem to achieve the
desired data read frequency. Enabling hrtimer to improve the
frequency characteristics.

Change-Id: I3358008dbc772445070d111e6da41caad9e0ff8b
Tracked-On: https://jira01.devtools.intel.com/browse/OAM-26476
Signed-off-by: Arun Radhakrishnan <arun.radhakrishnan@intel.com>
Reviewed-on: https://android.intel.com:443/486304
---
 arch/x86/configs/x86_64_defconfig | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/x86/configs/x86_64_defconfig b/arch/x86/configs/x86_64_defconfig
index 1dde633..a5b3cf7 100644
--- a/arch/x86/configs/x86_64_defconfig
+++ b/arch/x86/configs/x86_64_defconfig
@@ -3503,6 +3503,9 @@ CONFIG_STAGING=y
 #
 # Triggers - standalone
 #
+CONFIG_IIO_CONFIGFS=y
+CONFIG_IIO_HRTIMER_TRIGGER=y
+CONFIG_IIO_SW_TRIGGER=y
 # CONFIG_IIO_PERIODIC_RTC_TRIGGER is not set
 # CONFIG_IIO_SIMPLE_DUMMY is not set
 # CONFIG_FB_SM7XX is not set
-- 
1.9.1

